src/interface/Options/ToggleFilters.tsx

```diff
@@ -8,10 +8,11 @@ export default () => {
   const { filtersShown, setFiltersShown } = useInterfaceContext();
   const toggleFilters = () => setFiltersShown(!filtersShown);
   const { t } = useTranslation('filter');
+  const label = filtersShown ? t('hide') : t('show');
```

Comment:
Should be `hide filters` and `show filters`, otherwise, it makes no difference to the button content.

```suggestion
  const label = filtersShown ? t('hide filters') : t('show filters');
```